frontend/ui/widget/inputtext.lua

```diff
@@ -182,6 +182,31 @@ else
     function InputText:initEventListener() end
 end
 
+function InputText:onKeyPress(key)
+
+    -- logger.dbg(key)
+
+    if key["key"]:len() == 1 and not key["Ctrl"] and not key["Alt"] then
+
+        if key["modifiers"]["Sym"] then
```

Comment:
Hm no, apparently not. :-P

Reply:
Means it's just "A" or "Z" or something, as opposed to "Backspace" or something

Reply:
You might also want to take a look at the SDL headers/documentation. I don't know otoh if there's anything relevant in there, but you just might be trying to implement a difficult workaround for something that's a standard part of SDL.

Reply:
The `key` table looks like this:
```
{
    ["key"] = "F",
    ["F"] = true,
    ["modifiers"] = {
        ["Alt"] = false,
        ["Ctrl"] = false,
        ["Shift"] = false,
        ["Sym"] = false,
    },
}
```
Is there somewhere I can hook into a more complete object?

Reply:
Yes, in base (see the link I left in Gitter).

Reply:
Right, but how to get at that from InputText?

Reply:
Note how I moved some of the logic to front in the past because the quick touch event emulation in base was insufficient. You might have to do something similar. The reverse would be putting front logic in base, which while functional would be unsound.